src/index.tsx

```diff
@@ -106,11 +106,7 @@ export {default as IconError} from './icons/icon-error';
 export {default as IconSuccess} from './icons/icon-success';
 export {default as IconSuccessVivo} from './icons/icon-success-vivo';
 
-// Skeletons
-export {default as SkeletonText} from './skeletons/skeleton-text';
-export {default as SkeletonCircle} from './skeletons/skeleton-circle';
-export {default as SkeletonRow} from './skeletons/skeleton-row';
-export {default as SkeletonMedia} from './skeletons/skeleton-media';
+export * from './skeletons';
```

Comment:
I'd prefer to be explicit here and list all the exports, we could add a new export to skeletons in the future and want it to be used internally only, with the `*` we may end up making it public without noticing

Reply:
makes sense, alright!